### DISCUSSION TITLE
Could we come up with a standard maven naming convention?

### DISCUSSION THREAD

**Discussion by rhoegg** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#99] [Ideas]
I am finding different naming conventions for the various projects:

- `mule4-aichain-connector`
- `mac-web-crawler`
- `mac-whisperer`
- `mulechain-einstein1-connector`
- `mulechain-bedrock`
- `mulechain-vectors`
- `mac-inference-chain`

Historically, connectors have usually used the format `mule-<something>-connector` (see [repo1](https://repo1.maven.org/maven2/org/mule/connectors/) and [mulesoft repo](https://repository.mulesoft.org/nexus/content/repositories/releases/com/mulesoft/connectors/)) or sometimes `mule4-<something>-connector`.

My preference would be using the same standard, but I could also be OK with `mac-<something>-connector` if you all strongly prefer it.  Thoughts?

- `mule-aichain-connector` or `mule-langchain4j-connector`
- `mule-web-crawler-connector`
- `mule-whisperer-connector`
- `mule-einstein1-connector`
- `mule-aws-bedrock-connector`
- `mule-vectors-connector`
- `mule-inference-connector`

**Comment by rhoegg** [comment]
Also, yes I am volunteering to send the PRs :)

**Reply by amirkhan-ak-sf** [reply]
Lets agree on the following:

- `mule-aichain-connector` 
- `mule-web-crawler-connector`
- `mule-whisperer-connector`
- `mule-einstein1-connector`
- `mule-amazon-bedrock-connector`
- `mule-vectors-connector`
- `mule-inference-connector`

I will proceed with the renaming. Thanks Ryan!
